src/System.IO.Compression/tests/ZipArchive/zip_CreateTests.cs

```diff
@@ -149,6 +149,88 @@ public static void CreateUncompressedArchive()
             }
         }
 
+        [Theory]
+        [InlineData("minimalTwoFiles")]
+        public static void CreateNormal_VerifyDataDescriptor(string folder)
+        {
+            void CreateEntry(ZipArchive archive, string fileName, string fileContents)
+            {
+                ZipArchiveEntry entry = archive.CreateEntry(fileName);
+                using (StreamWriter writer = new StreamWriter(entry.Open()))
+                {
+                    writer.Write(fileContents);
+                }
+            }
+
+            string fileName = folder + ".zip";
```

Comment:
should this use `zfolder(folder)` somewhere? not sure

In general these tests seem to be assuming they control the current directory during test execution and can write to it. In UAP test runs at least, the tests do not have access to the current directory. So tests should derive from FileCleanupTestBsae and put things in TestDirectory. These tests derive indirectly from that but don't seem to use it. Perhaps I'm missing something. Anyway - not a problem to fix here, just a matter of following existing usage in the tests.

Reply:
> should this use `zfolder(folder)` somewhere? not sure

I was originally planning to add my own test data but the whole process was overkill in my opinion (See [FileData.cs](https://github.com/dotnet/corefx/blob/a10890f4ffe0fadf090c922578ba0e606ebdd16c/src/Common/tests/System/IO/Compression/FileData.cs#L31)) so I thought it would be faster and easier toadd the two text files directly with C# using ZipArchiveEntry objects.

> In general these tests seem to be assuming they control the current directory during test execution and can write to it. In UAP test runs at least, the tests do not have access to the current directory.

Thanks for letting me know. Let me run an automation job first, and if it fails due to access to that folder, I'll investigate how/where to save my generated zip file.

Reply:
Thanks for the suggestion. I'll switch everything to use MemoryStreams.